### PR TITLE
Section2 AND Section3 updates

### PR DESCRIPTION
**Pull request by JohanBester**
Hi Folks,

Here is my combined PR for updates to section2 and section3 of the curriculum.

I also fixed any content errors I found during class.

One of the things I have done is to try and simplify the language -- using simpler expressions and shorter sentence with less complicated language structure.

I have done this because in both the last 2 cohorts there are people that are Not strong in English but have a desire to learn to code.

Kindly please review and let me know what changes I should make.

### CODE REVIEW

**Review by matsinet — CHANGES_REQUESTED** [review]
@JohanBester I have added comments

File: Section02/2-Homework.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 ## Set-up your Homework repo on GitHub
 
-1. Use the [Savvy-Homework template repo](https://github.com/savvy-coders/savvy-homework) to _create a new GitHub repo for homework_ - name it something that indicates it is for **Savvy Coders Homework**
+1. Use the [Savvy-Coders Starter template repo](https://github.com/savvy-coders/savvy-starter) to _create a new GitHub repo for homework_ - name it something that indicates it is for **Savvy Coders Homework**
```
[the remark this answers is not in the document]
Comment (JohanBester) [reply Section02/2-Homework.md:20]: Excellent suggestion. Will make that change.

File: Section02/2.1-EssentialDevTools.md [hunk 2]
```diff
@@ -115,55 +129,62 @@ The result of this activity should produce a directory structure that looks more
...
+<br>
 
 ## npm
 
 ### Node Package Manager
 
 [Node Package Manager (npm)](https://www.npmjs.com/) is our dependency-management system.
 
-The Savvy Coders template repo includes a file named `package.json`. JSON stands for **J**ava**S**cript **O**bject **N**otation. A JSON file is not JavaScript, but is _formatted_ into key-value pairs like a JavaScript object. Inside `package.json` is a key named `devDependencies` which lists all the dependencies/packages necessary for our development environment. When we run `npm install`, it checks our `package.json` file and installs the needed packages from npmjs.com.
+**After cloning the Savvy Coders Starter template repo to your local machine, open it in VS Code.**
```
[the remark this answers is not in the document]
Comment (JohanBester) [reply Section02/2.1-EssentialDevTools.md:156]: True. Thank you Matt! WIll add as well.

File: Section02/2.2-JSdataTypeAndVariables.md [hunk 3]
```diff
@@ -19,68 +19,88 @@ When working on a project there are many cases where the developer asks for data
...
+- Because computers only understand explicit, formalized instructions, writing programs is "**tedious** and **frustrating**." Communicating with a computer will be a struggle, not only to break-down seemingly simple tasks into even more basic commands, but also to find formal errors. involving parentheses, brackets, braces, and other special characters not normally used in everyday communication.
 
 <br >
 
 ---
 
 <br >
 
-Over the next few weeks you will be exposed to a lot of new information and concepts. Please remember to breathe and to be patient with yourself. You will probably be bad at programming for a time, just like many musicians are bad at playing their instruments for a time.
+Over the next few weeks you will be exposed to a lot of new information and concepts. Please remember to breath and to be patient with yourself. You will probably be bad at programming for a time, just like many musicians are bad at playing their instruments for a time.
```
[the remark this answers is not in the document]
Comment (JohanBester) [reply Section02/2.2-JSdataTypeAndVariables.md:38]: Ooops. sorry my bad.

**Review by drod1107 — DISMISSED** [review]

**Review by cbrantley4 — DISMISSED** [review]
Looks great!

**Review by matsinet — APPROVED** [review]

**Review by mjwebb1993 — APPROVED** [review]

**Review by cbrantley4 — APPROVED** [review]
Merge it

**Pull request merged**